tests/test_acp_client_with_interrupt.py

```diff
@@ -0,0 +1,163 @@
+# Copyright AGNTCY Contributors (https://github.com/agntcy)
+# SPDX-License-Identifier: Apache-2.0
+import datetime
+from typing import List, Optional, TypedDict
+from unittest.mock import Mock
+
+from langgraph.checkpoint.memory import MemorySaver
+from langgraph.graph import END, START, StateGraph
+from langgraph.types import Command
+from pydantic import BaseModel
+
+import agntcy_acp
+from agntcy_acp import ApiClientConfiguration
+from agntcy_acp.langgraph.acp_node import ACPNode
+from agntcy_acp.models import (
+    RunCreateStateless,
+    RunInterrupt,
+    RunOutput,
+    RunStateless,
+    RunStatus,
+    RunWaitResponseStateless,
+)
+
+
+class InputSchema(BaseModel):
+    content: Optional[str] = None
+
+
+class OutputSchema(BaseModel):
+    messages: List[dict]
+
+
+class StateMeasures(TypedDict):
+    user_feedback: Optional[str] = None
+    input: Optional[InputSchema] = None
+    output: Optional[OutputSchema] = None
+
+
+def test_interrupt_acp_node(
+    default_api_key,
+    default_agent_id,
+    monkeypatch,
+):
+
+    agent_id = "bogus-agent-id"
+    init_run_id = "bugus-run-id"
+    monkeypatch.setenv("PHILOSOPHER_AGENT_HOST", "http://phil_agent")
+    monkeypatch.setenv(
+        "PHILOSOPHER_AGENT_API_KEY", f'{{"x-api-key": "{default_api_key}"}}'
+    )
+
+    output_state = OutputSchema(messages=[{"role": "human", "content": "Yes"}])
+
+    def mock_response_deserialize(
+        self,
+        response_data,
+        response_types_map=None,
+    ):
+
+        run = RunWaitResponseStateless(
+            run=RunStateless(
+                run_id=init_run_id,
+                agent_id=default_agent_id,
+                creation=RunCreateStateless(
+                    agent_id=default_agent_id, multitask_strategy="interrupt"
+                ),
+                status=RunStatus.INTERRUPTED,
+                created_at=datetime.datetime.now(),
+                updated_at=datetime.datetime.now(),
+            ),
+            output=RunOutput.from_dict(
```

Comment:
I think you can pass it the object rather than go through the dict conversion like:

```python
output=RunOutput(RunInterrupt( ... ))
```